addons/l10n_ro/models/res_company.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright (C) 2020 NextERP Romania (https://www.nexterp.ro) <contact@nexterp.ro>
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    company_registry = fields.Char(related="partner_id.nrc")
```

Comment:
It is creative, but it is not ideal.  It should not be readonly and it is not possible for other localizations this way to do the same hack.  (so I would prefer leaving this out)

Reply:
@jco-odoo Probably on v15.0...